### DISCUSSION TITLE
Seems to not work with feed from a personal website

### DISCUSSION THREAD

**Discussion by Tobi-De** [gautamkrishnar/blog-post-workflow#180] [Q&A]
Hey, thanks for this amazing work.
I'm trying to get this to work with my personal website, something I built myself from scratch, I have an rss feed but I get an error when I try to run the workflow, is this a format issue or something else? Thanks.

the feed url
https://oluwatobi.dev/blog/feed/

<img width="1265" alt="image" src="https://user-images.githubusercontent.com/40334729/200300037-bdf2ecaf-8730-4c1a-8034-bdfb313da660.png">

**Comment by gautamkrishnar** [comment]
@Tobi-De maybe it's your backend returning an empty response initially when called from github IP

edit: yes it is:
![Screenshot 2022-11-07 at 6 02 35 PM](https://user-images.githubusercontent.com/8397274/200311457-1955e663-4f4a-4260-9579-edc622770279.png)

**Reply by Tobi-De** [reply]
Oh, I see, weird
I've updated the rss feed url 

https://oluwatobi.dev/blog/rss.xml/

**Reply by Tobi-De** [reply]
name: Latest blog post workflow
on:
  schedule: # Run workflow automatically
    - cron: '0 * * * *' # Runs every hour, on the hour
  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the GitHub Actions Workflow page directly

jobs:
  update-readme-with-blog:
    name: Update this repo's README with latest blog posts
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Pull in my posts from my personal site
        uses: gautamkrishnar/blog-post-workflow@v1
        with:
          feed_list: "https://oluwatobi.dev/blog/rss.xml/"

**Reply by Tobi-De** [reply]
@gautamkrishnar 
I also updated the github workflow.
I think it was missing the trailing slash.
It still doesn't work, I don't understand why from gihtub IP it returns an empty page.

**Reply by Tobi-De** [reply]
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/40334729/200324809-a0c1ed24-ddb9-40b1-9694-654499e1530f.png">

The workflow still fails, even if the content can be fetch correctly this time.
